dotnet/src/Connectors/Connectors.AI.OpenAI/TextToImage/AzureOpenAITextToImageResponse.cs

```diff
@@ -3,19 +3,19 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Text.Json.Serialization;
 
-namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.ImageGeneration;
+namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.TextToImage;
 
 /// <summary>
-/// Image generation response
+/// Text to image response
 /// </summary>
 [Experimental("SKEXP0012")]
-public class AzureOpenAIImageGenerationResponse
+public class AzureOpenAITextToImageResponse
 {
     /// <summary>
-    /// Image generation result
+    /// Text to image result
     /// </summary>
     [JsonPropertyName("result")]
-    public ImageGenerationResponse? Result { get; set; }
+    public TextToImageResponse? Result { get; set; }
```

Comment:
nit. Response within another response, sounds confusing, can work on that after the content goes in... 

AzureOpenAI`TextToImageResponse`
    public `TextToImageResponse` Result
    
```suggestion
    public TextToImageResponse? Result { get; set; }
```